src/mappings/governance.ts

```diff
@@ -65,6 +65,7 @@ export function handleProposalSubmitted(event: ProposalSubmitted): void {
 export function handleProposalVoteSubmitted(event: ProposalVoteSubmitted): void {
   let proposalId = event.params._proposalId.toString()
   let proposal = Proposal.load(proposalId)
+  if (proposal === null) return
```

Comment:
if we vote on a proposal that doesn't exist today, does that halt the indexer? i guess it's probably not possible because of the way the smart contract is written, it would prevent that

Reply:
I believe returning here would just be a noop on the indexer, if someone proposed a vote on a proposal that doesn't exist then there's nothing to do